src/components/App.routes.tsx

```diff
@@ -24,7 +24,7 @@ import ListingsResult from './routes/Listing/ListingsResult';
 import Login from './routes/authentication/Login';
 import Logout from './routes/authentication/Logout';
 import SignUp from './routes/authentication/SignUp';
-import { StripeExpressComplete, StripeExpressNew } from './routes/Account/StripeExpress';
+import { StripeConnect, StripeExpressComplete, StripeExpressNew } from './routes/Account/StripeExpress';
```

Comment:
maybe it's time to clean up the naming here to make it more clear: (my mistake since i came up with it originally)

Stripe Connect is the umbrella program
* Stripe Express and Stripe Standard are subset.
* `StripeExpressComplete` is used to handle completion of both new stripe express accounts and connecting existing stripe standard accounts.

what if we change to:
```
/account/stripe_connect/new_express_account => /routes/Account/Stripe/StripeConnectExpressNew
/account/stripe_connect/link_existing => /routes/Account/Stripe/StripeConnectLinkExisting
/account/stripe_connect/complete => /routes/Account/Stripe/StripeConnectComplete
```

Reply:
Agreed; okay if I do that in another PR to keep functional changes and refactors separate?

Also, what do you think of:

    /account/stripe/new → /routes/Account/Stripe/StripeNew
    /account/stripe/link → /routes/Account/Stripe/StripeLink
    /account/stripe/complete → /routes/Account/Stripe/StripeComplete

Rationale is to avoid saying "connect" or "express" and instead treat these as implementation details. (Also just word golfing.)

Reply:
yeah, sounds good. this is also clear.